macOS 无法初始化 CEF

### 问题描述

_No response_

### 复现步骤

_No response_

### Ani 版本号

5.0.0-alpha03

### 操作系统

macOS (M 系列芯片)

### 应用日志

```
2025-08-16 14:08:25,836 [INFO ] Ani: [JCEF init] anitorrent loaded
2025-08-16 14:08:25,837 [INFO ] Ani: [JCEF init] Calling AniCefApp.initialize
2025-08-16 14:08:25,843 [INFO ] app.platform.AniCefApp: Absolute path: /
2025-08-16 14:08:25,843 [INFO ] app.platform.AniCefApp: user.dir: /
2025-08-16 14:08:25,843 [INFO ] app.platform.AniCefApp: Java home: /Applications/Ani.app/Contents/runtime/Contents/Home
2025-08-16 14:08:25,843 [INFO ] app.platform.AniCefApp: Found CEF framework at /Applications/Ani.app/Contents/runtime/Contents/Home/lib/Frameworks
2025-08-16 14:08:25,847 [WARN ] ani-root: Uncaught exception in coroutine [me.him188.ani.app.platform.CommonKoinModuleKt$createAppRootCoroutineScope$$inlined$CoroutineExceptionHandler$1@733af2d7, StandaloneCoroutine{Cancelling}@43143054, Dispatchers.Default]
java.lang.ExceptionInInitializerError
	at me.him188.ani.app.platform.AniCefApp.createCefApp(AniCefApp.kt:146)
	at me.him188.ani.app.platform.AniCefApp.initialize(AniCefApp.kt:204)
	at me.him188.ani.app.desktop.AniDesktop$main$9.invokeSuspend(AniDesktop.kt:315)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:586)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:829)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:717)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:704)
Caused by: java.lang.NullPointerException: Cannot invoke "java.io.File.getAbsolutePath()" because "<local3>" is null
	at jcef/com.jetbrains.cef.remote.NativeServerManager.getServerExe(Unknown Source)
	at jcef/com.jetbrains.cef.remote.NativeServerManager.isRemoteSupported(Unknown Source)
	at jcef/org.cef.CefApp.isRemoteSupported(Unknown Source)
	at jcef/org.cef.CefApp.<clinit>(Unknown Source)
	... 9 more
```

```2025-08-16 16:43:34,927 [INFO ] DesktopWebMediaResolver: Final config: WebViewConfig(cookies=[quality=1080P])
2025-08-16 16:43:34,930 [WARN ] WebViewVideoExtractor: AniCefApp isn't initialized yet.
2025-08-16 16:43:34,930 [INFO ] HttpClientProvider:   GET https://mikanani.me/home/search?searchstr=%E6%8B%94%E4%BD%9C%E5%B2%9B : 200  in 762.555625ms
2025-08-16 16:43:34,930 [WARN ] PlayerSession: Failed to resolve video source due to VideoSourceResolutionException
java.lang.IllegalStateException: Failed to resolve video source due to VideoSourceResolutionException
        at me.him188.ani.app.domain.episode.PlayerSession$loadMedia$2.invokeSuspend(PlayerSession.kt:110)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.UndispatchedCoroutine.afterResume(CoroutineContext.kt:266)
        at kotlinx.coroutines.AbstractCoroutine.resumeWith(AbstractCoroutine.kt:101)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
        at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:586)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:829)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:717)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:704)
Caused by: me.him188.ani.app.domain.media.resolver.MediaResolutionException: Failed to resolve video source: NO_MATCHING_RESOURCE
        at me.him188.ani.app.domain.media.resolver.DesktopWebMediaResolver$resolve$2.invokeSuspend(DesktopWebMediaResolver.kt:115)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        ... 5 more
2025-08-16 16:43:34,931 [INFO ] PlayerLoadErrorHandler: Player errored, automatically switching to next media```

--- a/app/desktop/build.gradle.kts
+++ b/app/desktop/build.gradle.kts
@@ -225,20 +225,21 @@ afterEvaluate {
         }
 
         Os.MacOS -> {
-            tasks.named("createRuntimeImage", AbstractJLinkTask::class) {
+            tasks.named("createReleaseDistributable", AbstractJPackageTask::class) {
                 val dirsNames = listOf(
                     // From your (JBR's) Java Home to Packed Java Home 
-                    "../Frameworks" to "lib/",
+                    "../Frameworks" to "Contents/runtime/Contents",
                 )
 
                 dirsNames.forEach { (sourcePath, destPath) ->
                     val source = File(javaHome.get()).resolve(sourcePath).normalize()
                     inputs.dir(source)
-                    val dest = destinationDir.file(destPath)
-                    outputs.dir(dest)
                     doLast("copy $sourcePath") {
+                        val appBundle = destinationDir.get().asFile.walk().find { it.name.endsWith(".app") && it.isDirectory }
+                        var dest = appBundle?.resolve(destPath)?.normalize()
+                        ?: throw GradleException("Cannot find .app bundle in $appBundle")
                         ProcessBuilder().run {
-                            command("cp", "-r", source.absolutePath, dest.get().asFile.normalize().absolutePath)
+                            command("cp", "-r", source.absolutePath, dest.absolutePath)
                             inheritIO()
                             start()
                         }.waitFor().let {

